Dynavity/Dynavity/persistence/data-mapping/elements/uml-shapes/DiamondUmlElementDTO.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+struct DiamondUmlElementDTO: UmlElementProtocolDTO, Mappable {
+    // For storing UML connectors
+    let id: UUID
+
+    let canvasProperties: CanvasElementPropertiesDTO
+
+    let label: String
+    let umlType: String
+    let umlShape: String
+
+    init(model: DiamondUmlElement) {
+        self.id = UUID()
```

Comment:
Does the id need to be the constant for the same connector across different reads / writes to the storage?

Reply:
Nope! If we wanted to do that though, we probably need to have it as part of the model?